autoscheduler/frontend/src/components/App/testMeetings.ts

```diff
@@ -0,0 +1,106 @@
+import Section from '../../types/Section';
+import Instructor from '../../types/Instructor';
+import Meeting, { MeetingType } from '../../types/Meeting';
+
+// DEBUG
+const testSection1 = new Section({
+  id: 123456,
+  subject: 'SUBJ',
+  courseNum: 234,
+  sectionNum: 500,
+  minCredits: 3,
+  maxCredits: null,
+  currentEnrollment: 56,
+  instructor: new Instructor({
+    id: 123456,
+    name: 'Aakash Tyagi',
```

Comment:
Going to be an upcoming change in #94, but the instructor's id/primary key is now their name (recent Banner update changed the data), so we'll want to update the Instructor type accordingly